documentation/examples/cats/full.ts

```diff
@@ -121,10 +121,10 @@ const CatSchema = coda.makeObjectSchema({
       type: coda.ValueType.String,
       codaType: coda.ValueHintType.DateTime,
     },
-    id: { type: coda.ValueType.String },
+    catId: { type: coda.ValueType.String },
   },
   displayProperty: "image",
-  idProperty: "id",
+  idProperty: "catId",
   featuredProperties: ["tags"],
   identity: {
```

Comment:
Looks like we can delete this?

Reply:
This was me trying to fix something unrelated in the same PR :-) I can remove for focus.